snyk-monitor-operator-source.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
```

Comment:
Should this yaml in some folder in `test/fixtures`?

Reply:
Checked in by accident 🙏